libraries/c_sdk/standard/mqtt/src/iot_mqtt_operation.c

```diff
@@ -563,8 +563,7 @@ void _IotMqtt_DestroyOperation( _mqttOperation_t * pOperation )
         IotLogDebug( "(MQTT connection %p, %s operation %p) Removed operation from connection lists.",
                      pMqttConnection,
                      IotMqtt_OperationType( pOperation->u.operation.type ),
-                     pOperation,
-                     pMqttConnection );
+                     pOperation );
```

Comment:
These probably needs to go the CSDK I will add this work item to my team. Thanks!

Reply:
It is already fixed in CSDK.

Reply:
Oh nvm I saw the same problem on line 745.